src/lib/stores/userStore.ts

```diff
@@ -0,0 +1,16 @@
+import { writable } from 'svelte/store';
+
+import { User } from '$lib/types/user';
```

Comment:
We are encountering an error during `svelte-check` due to the ts config options. These options enforce a clear distinction between types and runtime values, requiring that types be imported using a type-only import. 

`import type { User } from '$lib/types/user';`